models/model.js

```diff
@@ -53,7 +54,15 @@ exports.getCommentsById = (article_id) => {
       if (rows.length) {
         return rows;
       } else {
-        return Promise.reject({ status: 404, msg: "bad request" });
+        return db
```

Comment:
I'm not sure what we're trying to do here? If there are no topics returned you are doing a query for all articles but in the .then you are sending back only rows[0] - i.e. the first article?